docker-compose.yml

```diff
@@ -223,5 +223,44 @@ services:
       context: runner
       dockerfile: regular-jdk/Dockerfile
       args:
-        - UNMAZEDBOOT_INTERNAL_BASE_IMAGE=sgrio/java-oracle:jdk_11_alpine
+        - UNMAZEDBOOT_INTERNAL_BASE_IMAGE=sgrio/java:jdk_11_alpine
```

Comment:
Some on those changes... Create a separate PR to fix specific atomic changes... After removing those from the PR, the PR would be clean for tests

Reply:
Sure, let me create a separate PR